data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -1718,6 +1718,19 @@ spec:
                       the subnets. Leave the hosted zone unset to have the installer
                       create the hosted zone on your behalf.
                     type: string
+                  lbType:
+                    description: 'LBType allows user to set a load balancer type.
+                      When this field is set the default ingresscontroller will get
+                      created using the specified LBType. If this field is not set
+                      then the default ingress controller of LBType Classic will be
+                      created. Valid values are: * "Classic": A Classic Load Balancer
+                      that makes routing decisions at either   the transport layer
+                      (TCP/SSL) or the application layer (HTTP/HTTPS). See   the following
+                      for additional details:   https://docs.aws.amazon.com/AmazonECS/latest/developerguide/load-balancer-types.html#clb
+                      * "NLB": A Network Load Balancer that makes routing decisions
+                      at the   transport layer (TCP/SSL). See the following for additional
+                      details:   https://docs.aws.amazon.com/AmazonECS/latest/developerguide/load-balancer-types.html#nlb'
+                    type: string
```

Comment:
Nit: Some double spaces in the description fileld.

Reply:
Hmm I see that, but I think this is generated from https://github.com/openshift/installer/pull/6478/files#diff-7941d8da46384fc43b6eeae7d5c3319065b16940457822346c9ccb08cbcf47b4R66, which uses a hanging indent and a bulleted list.

Should I remove the hanging indent and extra spaces from there?

Reply:
I propose we do that as a follow-up.